test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -1244,60 +1248,99 @@ public final void testBlockLoaderFromColumnReaderWithSyntheticSource() throws IO
         testBlockLoader(true, true);
     }
 
-    // Removed 'final' to silence this test in GeoPointFieldMapperTests, which does not support synthetic source completely
-    public void testBlockLoaderFromRowStrideReaderWithSyntheticSource() throws IOException {
+    public final void testBlockLoaderFromRowStrideReaderWithSyntheticSource() throws IOException {
         testBlockLoader(true, false);
     }
 
-    protected boolean supportsColumnAtATimeReader(MapperService mapper, MappedFieldType ft) {
-        return ft.hasDocValues();
+    /**
+     *  Should the doc-values reader be used instead of reading from source.
```

Comment:
The javadoc looks out of date.

Reply:
Updated, and removed the rowStrideReader boolean, since that was always true. I had added that expecting to need it in the spatial tests, but did not use it.